source/install/install-ubuntu-1404.rst

```diff
@@ -0,0 +1,23 @@
+.. _install-ubuntu-1404:
+
+=========================================
+Installing Mattermost on Ubuntu 14.04 LTS
+=========================================
+
+Install a production-ready Mattermost system on 1 to 3 machines.
+
+.. include:: install-common-intro.rst
+
+.. contents:: Install and configure the components in the following order. Note that you need only one database, either MySQL or PostgreSQL.
+  :backlinks: top
+  :local:
+  
+.. include:: install-ubuntu-1404-server.rst
+.. include:: install-ubuntu-1404-mysql.rst
```

Comment:
I feel like the mysql and postgresql should be under a subsection for Database Server? Rather than putting the note: "Note that you need only one database, either MySQL or PostgreSQL."